interface/libyami.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LIBYAMI_H_
+#define LIBYAMI_H_
+
+#include <YamiVersion.h>
+
+#include <VideoCommonDefs.h>
+
+#include <VideoDecoderDefs.h>
+#include <VideoDecoderHost.h>
```

Comment:
but the libyami local file also include those header files, so it will have to scan the installed path when compiling libyami if their include are all <>.